src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -147,6 +155,9 @@ export class SkyLookupComponent
     return this._selectMode || SkyLookupSelectMode.multiple;
   }
 
+  @Output()
```

Comment:
This will need a JSDoc description.

Reply:
I've fixed this and also tweaked the wording of the JSDoc descriptions.